### PR TITLE
feat(layers-ordering): visualize layers order

### PR DESCRIPTION
**Pull request by alebinson**

### CODE REVIEW

File: src/common/components/grid/grid.tsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ export interface IGridRowDataDetailsExt {
   fullWidth: boolean;
   isVisible: boolean;
 };
+export interface GridRowNode extends RowNode {};
```
Comment (asafMasa) [inline src/common/components/grid/grid.tsx:43]: do you think IGridRowNode is better (add the I before)

File: src/discrete-layer/components/layers-results/layers-results.tsx [hunk 2]
```diff
@@ -16,6 +16,8 @@ interface LayersResultsComponentProps {
 
 const pagination = true;
 const pageSize = 10;
+const emidiateExecution = 0;
```
Comment (asafMasa) [inline src/discrete-layer/components/layers-results/layers-results.tsx:19]: immediate

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/common/components/grid/grid.tsx (hunk 1 above)
Comment (alebinson) [reply src/common/components/grid/grid.tsx:43]: Aaaaa... those, I keep them as much as close(naming convention) to original one(s)[just prefix them with "**Grid**"] there is a lot of such "interfaces".

Let's leave it this way.

File: src/discrete-layer/components/layers-results/layers-results.tsx (hunk 2 above)
Comment (alebinson) [reply src/discrete-layer/components/layers-results/layers-results.tsx]: "Eagle eye" - fixed

**Review by asafMasa — APPROVED** [review]

**Pull request merged**